### ISSUE TITLE
Request: Prevent motion from sensing an event at boot/daemon restart

### ISSUE THREAD

**Issue by michaelfisher** [Motion-Project/motion#541]
It would be helpful for motion to not record immediately upon starting. Would it be possible to have motion grab a few frames for comparison before starting detection? This is assuming that the reason it records right off the bat is that the 1st image it gets is "different" from the non-image it had at start. This behavior is problematic because the on_event_start script, which contains smart home commands, gets triggered falsely anytime the machine reboots (via cronjob). I'd happily accept using the "-m" command line option if someone could please tell me how to have the daemon run with that option.

**Comment by tosiara** [comment]
Usually I don't get recording started immediately after start. It may happen sometimes, but I tend to blame my USB camera which may adjust exposition at runtime when device is opened. Never investigated that further
If it happens for you, try running motion in setup mode with motion video recording and you will see which area and why is triggering motion

Another suggestion that came from user mail list is start motion with disabled detection, wait few seconds and then enable detection using http api call.

**Comment by michaelfisher** [comment]
I'm running motion on a RasPi, with the RasPi camera module. Using the setup mode, I notice it may be the exposure adjusting as the camera starts up. With that in mind, I'd love to use the "start motion with disabled detection" option. I've got scripts built to make the API calls to start/stop detection based on conditions I've defined. Can you explain how to start the motion daemon with detection disabled? The "-m" option only applies when starting motion from the command line. There is currently no option to disable detection on startup in the config file.

**Comment by michaelfisher** [comment]
I think I have solved this issue, though I think of it as a workaround. Within /etc/init.d/motion, in the "start" section, find the "start-stop-daemon" line. Just before the semicolon, add "-- -m" (without the quotes). Anything after the "--" is passed as an argument to the command being started. This is persistent across reboots and daemon restarts. I apologize for clogging up the issues because I failed to RTFM long enough. However, I'd still like to see this added as a config file option. Thanks for all your hard work on this project!
